app/Listeners/TaskUpdateXP.php

```diff
@@ -26,66 +26,64 @@ public function __construct()
     /**
      * Handle the event.
      *
-     * @param  TaskUpdate $event
+     * @param  ModelUpdate $event
      * @return void
      */
-    public function handle(TaskUpdate $event)
+    public function handle(ModelUpdate $event)
     {
-        // I don't always make PRs, but when I do I make one with every code change I have, even work in progress
-        return $this;
-        $this->tasks = $event->tasks;
+        $this->model = $event->model;
 
         //task user id
-        $user_id = $this->tasks->task_history[0]['user'];
+        $userId = $this->model->task_history[0]['user'];
 
         //project owner id
-        $project_owner_id = $this->tasks->owner;
+        $projectOwnerId = $this->model->ownerId;
 
-        //get task's XP value
-        $task_xp = $this->tasks->xp;
+        $userProfile = Profile::find($userId);
+        $ownerProfile = Profile::find($projectOwnerId);
```

Comment:
`$projectOwnerId` is task owner (person that created the task, not even assigned person), we need project owner